src/Coordination/KeeperStateManager.cpp

```diff
@@ -321,6 +323,11 @@ KeeperStateManager::KeeperStateManager(
     , keeper_context(keeper_context_)
     , logger(getLogger("KeeperStateManager"))
 {
+    if (keeper_context_->getCoordinationSettings()[CoordinationSetting::rotate_log_storage_interval] <= 0)
```

Comment:
Just change the type of the `rotate_log_storage_interval` in `CoordinationSettings.cpp` to NonZeroUInt64.

Reply:
Just do this.

Reply:
Done. I didn't know that was an option :).